File: Module.Frontend.TPM/Client/model/tpm/sfatype/HistoricalSFAType.js
Ext.define('App.model.tpm.sfatype.HistoricalSFAType', {
    extend: 'Ext.data.Model',
    idProperty: '_Id',
    breezeEntityType: 'SFAType',
    fields: [
        { name: '_Id', type: 'string', hidden: true },
        { name: '_ObjectId', hidden: true },
        { name: '_User', type: 'string', isDefault: true },
        { name: '_Role', type: 'string', isDefault: true },
        { name: '_EditDate', type: 'date', isDefault: true, timeZone: +3, convert: dateConvertTimeZone },
        { name: '_Operation', type: 'string', isDefault: true },
        { name: 'Name', type: 'string', hidden: false, isDefault: true }
    ],
    proxy: {
        type: 'breeze',
        resourceName: 'HistoricalSFATypes',
        reader: {
            type: 'json',
            totalProperty: 'inlineCount',
            root: 'results'
        }
    }
});

File: Module.Persist.TPM/Model/History/HistoricalSFAType.cs
using Core.History;
using Module.Persist.TPM.Model.TPM;
using System;

namespace Module.Persist.TPM.Model.History
{
    [AssociatedWith(typeof(SFAType))]
    public class HistoricalSFAType : BaseHistoricalEntity<System.Guid>
    {
        public string Name { get; set; }

    }
}
